config/webpack/environment.js

```diff
@@ -43,6 +44,13 @@ environment.loaders.append('yaml', {
   type: 'json'
 })
 
+// Handle images coming from the assets folder
+environment.plugins.append('Copy', new CopyWebpackPlugin({
+  patterns: [
+    { from: 'app/assets/images', to: 'images' }
+  ]
+}));
+
```

Comment:
Doesn't this duplicate the images again?

Instead of copying, is there a way to tell webpack to map `images/` to `assets/images/`, and avoid the copy?